### DISCUSSION TITLE
Wallet goes "not syncronized" sometimes

### DISCUSSION THREAD

**Discussion by YahorL** [Chia-Network/chia-blockchain#4725] [Support]
Hello, I've noticed that my wallet sometimes goes "not synchronized" and it happens randomly a few times during the day. After a while the wallet get synchronized, but it can be an hour which affects my mining. Any ideas what is happening? Maybe it because I updated recently to 1.1.5?

**Comment by markusmazurczak** [comment]
Hi. It seems that this issue affects a lot of people. There are many many open bugs for this issue but I think this is the one with the most attention, at least from the community: https://github.com/Chia-Network/chia-blockchain/issues/3298

I dont know what is the problem. The logs (at INFO level) do not give a hint. The connections just get drop.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
